### ISSUE TITLE
[FR]: Beta channel request

### ISSUE THREAD

**Issue by sbias** [MTrab/landroid_cloud#189]
can we get the 2.1 versions in a beta channel and 2.0 in stable?
for sure, i will use the beta, but there are 2 thinks. If i didn't started with the 2.0 version which was working out-of-the-box, i know, i would gave up already. And the other, hacs is only showing the last 5 releases - so, without manual work there is currently no way back to that version that works for me.

would be awesome to think about this.
and thanks for this awesome integration :)

**Comment by MTrab** [comment]
Yeah - was actually thinking of mowing to a release plan where even numbers were stable and uneven were beta :)
